api/api.yaml

```diff
@@ -47,6 +47,24 @@ paths:
       tags:
         # Put in facets_controller.py instead of default_controller.py.
         - Facets
+  /exportUrl:
+    # This is post because this is not idempotent; a GCS file is created.
+    post:
+      description: >
+        Gets a signed URL to a GCS zip file of JSON files. The JSON files
+        represent entites to be exported to a Saturn workspace.
```

Comment:
I'd elaborate on "represent entities" to mention that specifically, the JSON contains CreateEntity requests as defined by the Saturn API.

Reply:
Done. It's not "Saturn API" -- I specifically mention https://rawls.dsde-prod.broadinstitute.org/#!/entities/create_entity to clarify the exact API.